### ISSUE TITLE
Concern about the performance

### ISSUE THREAD

**Issue by yarkable** [ArchipLab-LinfengZhang/Object-Detection-Knowledge-Distillation-ICLR2021#7]
Hi, thanks for your great work. Following your supplementary materials, I use RetinaNet-ResNet101 to distill RetinaNet-ResNet50, and I plot mAP curve as follows, which can be seen that performance of KD method is higher than student, but it raise slowly in the later training process, and nearly no gain compared to student model. I wonder if it's a common phenomenon? Or there is sth wrong? 

![图片](https://user-images.githubusercontent.com/33142987/132116458-77ab0666-878a-47dc-874b-fb0dff25f233.png)

**Comment by ArchipLab-LinfengZhang** [comment]
Hi yarkable 
Maybe you can try train detectors for more epochs. Usually, I find that KD performs much better for 2X times training (24 epochs).

**Comment by yarkable** [comment]
Yep, I back to see your paper, and it seems that you use 2x schedule in all your experiments. Is it a common practice for distillation in object detection?

**Comment by lji72** [comment]
@yarkable  Do you get the expected gain with 2X times training？

**Comment by yarkable** [comment]
@lji72 Haven't tried yet.
